yasp.config.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+// there is no possible way this is the right place for this logic...
+// TODO - put this shit in the right place
+const matchColumns = [{
+  displayName: 'ID',
+  field: 'match_idDisplay',
+  width: 2,
+  sorted: false,
+}, {
+  displayName: 'Hero',
+  field: 'hero_idDisplay',
+  width: 2,
+  sorted: false,
+  component: (srcUrl) => <img src={srcUrl} style={{ height: '24px' }} role="presentation" />,
+}, {
+  displayName: 'W/L',
+  field: 'radiant_winDisplay',
+  width: 1,
+  sorted: false,
+}, {
+  displayName: 'Mode',
+  field: 'game_modeDisplay',
+  width: 2,
+  sorted: false,
+}, {
+  displayName: 'Date',
+  field: 'start_timeDisplay',
+  width: 2,
+  sorted: false,
+}, {
+  displayName: 'Duration',
+  field: 'durationDisplay',
+  width: 2,
+  sorted: false,
+}, {
+  displayName: 'Kills',
+  field: 'killsDisplay',
+  width: 2,
+  sorted: false,
+}, {
+  displayName: 'Deaths',
+  field: 'deathsDisplay',
+  width: 2,
+  sorted: false,
+}, {
+  displayName: 'Assists',
+  field: 'assistsDisplay',
+  width: 2,
+  sorted: false,
+}];
+
+// there is no possible way this is the right place for this logic...
+// TODO - put this shit in the right place
+const playerMatchTransformSwitch = (match, field) => {
+  switch (field) {
+    case 'hero_id':
+      return 'https://yasp.co/apps/dota2/images/heroes/pudge_full.png';
+    case 'radiant_win':
+      return match.player_slot < 64 && match[field] || match.player_slot >= 64 && !match[field] ? 'W' : 'L';
+    case 'game_mode':
+      return match[field] === 22 ? 'Ranked All Pick' : 'All Pick';
```

Comment:
see `json` directory in yasp core for mappings.  Should we have the server return them, or bundle them into the webpack build?